src/app/ApplicationTemplate.Tests/TestBase.cs

```diff
@@ -15,14 +18,15 @@ namespace ApplicationTemplate.Tests
 	public class TestBase : IAsyncLifetime
 	{
 		protected static readonly CancellationToken DefaultCancellationToken = CancellationToken.None;
+		protected static readonly CancellationToken AnyCancellationToken = It.IsAny<CancellationToken>();
```

Comment:
why we need to have this?

Reply:
Good point, you will need it once you start mocking. (e.g. MyService.GetCurrent(AnyCancellationToken)). It tells the mocking engine that any cancellation token can be used.